docs/build/tutorials/platform/adjusting-gas-price-during-high-network-activity.md

```diff
@@ -0,0 +1,110 @@
+---
+description: This tutorial will help users to adjust their priority fee and max fee cap during high network activity and take advantage of the benefits of dynamic fee transactions.
+---
+
+# Adjusting Gas Price During High Network Activity
+
+Sometimes during periods of high network activity, transactions either remain pending for a very long duration or instantly get a failed transaction notification. This may cause panic among users if they can't figure out what went wrong!
+
+## Probable Reasons You are Here
+
+* Your transaction has stalled, and you don't know what to do
+* Your transaction has failed, with an error - `transaction underpriced`
+* It's your first transaction, and you want to be sure about any potential issues
+* Just for general knowledge on adjusting dynamic fee settings
+
+If these are your reasons for being here, then you can either go through this entire section, for a better understanding of the scenario or directly skip to the [solution](#adjusting-gas-fees-before-submitting-the-transaction).
+
+## Good to Know Keywords and Concepts
+
+The amount of computation used by a transaction is measured in units of `gas`. Each unit of gas is paid for in AVAX at the `gas price` for the transaction. The `gas price` of the transaction is determined by the parameters of the transaction and the `base fee` of the block that it is included in.
+
+To avoid draining the user's wallet due to using computation infinitely, transactions are submitted with a `gas limit`, which denotes the maximum units of gas that a particular transaction is allowed to consume.
```

Comment:
```suggestion
To avoid draining the user's wallet due to non-terminating execution through the EVM, transactions are submitted with a `gas limit`, which denotes the maximum units of gas that a particular transaction is allowed to consume.
```